src/3rd_party.h

```diff
@@ -0,0 +1,44 @@
+#ifndef __THIRD_PARTY_H
+#define __THIRD_PARTY_H
+
+/**
+  * @file
+  * @brief Functions to call for third_party_main
+  */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "swupd.h"
+
+/**
+ * @brief Creates a repo folder under THIRDPARTY_REPO_PREFIX 
+ * if it does not exist
+ *
+ * @return 0 on success, SWUPD_THIRD_PARTY_REPO_EXISTS if it already exists
+ * and negative numbers on errors
+ */
+enum swupd_code third_party_add(int argc, char **argv);
```

Comment:
Rename all functions that are a main to add a _main suffix. This one should be `third_party_add_main`

Reply:
Makes sense will change in next push